content/docs/hello-world.md

```diff
@@ -8,6 +8,7 @@ next: introducing-jsx.html
 
 The smallest React example looks like this:
 
+TODO - this code sample does not work, even after fixing the 'Hello, World' line to be enclosed in quotes, Codepen gives an error that "ReactDOM is not defined"
```

Comment:
Throughout the documentation import statements are not present in the shorter code snippets. This can be fixed by importing `ReactDOM`.